language/move-prover/tests/sources/functional/inconsistency.move

```diff
@@ -0,0 +1,37 @@
+module Inconsistency {
```

Comment:
Do you have a test running the prover with --check-inconsistency?  For this test, I suppose that there would be a .exp file.